icu4c/source/i18n/unicode/messageformat2.h

```diff
@@ -0,0 +1,138 @@
+// © 2016 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#ifndef U_HIDE_DEPRECATED_API
+
+#ifndef MESSAGEFORMAT2_H
+#define MESSAGEFORMAT2_H
+
+#if U_SHOW_CPLUSPLUS_API
+
+/**
+ * \file
+ * \brief C++ API: Formats messages in a language-neutral way using the draft MessageFormat 2.0.
```

Comment:
what does "language-neutral" mean in this context? Do you mean "programming language" here? If you mean "natural language", kind of the point of MF is that it performs locale-sensitive selection and formatting... 😄

Reply:
I copied/pasted this comment from https://github.com/unicode-org/icu/blob/main/icu4c/source/i18n/unicode/msgfmt.h#L26 because I wasn't sure what else to put there... but maybe it's not necessary anyway for now?